core/src/main/java/io/dekorate/processor/AptReader.java

```diff
@@ -53,24 +56,6 @@ public class AptReader implements SessionReader, WithProject {
     this.processingEnv = processingEnv;
     this.inputFileIncludePattern = Pattern.compile(INPUT_FILE_INCLUDE_REGEX);
     this.inputFileExcludePattern = Pattern.compile(INPUT_FILE_EXCLUDE_REGEX);
-    hack();
-  }
-
-  private void hack() {
```

Comment:
I'm not 100% sure that this is indeed no longer needed. (let's see what the CI says about this).